stacks/upload-api-stack.js

```diff
@@ -28,6 +28,7 @@ export function UploadApiStack({ stack, app }) {
   const pkg = getApiPackageJson()
   const git = getGitInfo()
   const privateKey = new Config.Secret(stack, 'PRIVATE_KEY')
+  const uploadApiDid = new Config.Parameter(stack, 'UPLOAD_API_DID', { value: customDomain ? `did:web:${customDomain.domainName}` : '' })
```

Comment:
Can you also add to README?

Reply:
Yep, done :)

Reply:
https://github.com/web3-storage/w3infra/pull/91/commits/a52ff18f1b2ec03fd80af116b958a821e0f4fa2c